frontend/app/components/wp-edit/wp-edit-field/wp-edit-field.directive.html

```diff
@@ -1,4 +1,5 @@
 <div class="wp-edit-field inplace-edit"
+     ng-if="vm.workPackage"
```

Comment:
I've moved up the ng-if since the display-attr tries to directly access the work package, which may not yet be present. However, we can modify the form below to `ng-show`, which appears to be just as fast.